controllers/articles.controller.js

```diff
@@ -13,6 +14,20 @@ function getArticleById(request, response, next) {
     })
 }
 
+function postComment(request, response, next) {
+    const {article_id} = request.params;
+    const {body} = request;
+    
+    const promises = [selectArticle(article_id), insertComment(article_id, body)];
+    return Promise.all(promises)
+    .then((promisesResults) => {
+        response.status(201).send({postedComment: promisesResults[1]})
+    })
+    .catch((err) => {
+        next(err)
+    })
+}
```

Comment:
Lovely destructuring here and smart use of Promise.all to check that the entry exists and post the comment.